src/api/store/community.py

```diff
@@ -1,16 +1,206 @@
-from database.models import Community, CommunityMember, UserProfile, Action, Event, Graph, Media, ActivityLog, AboutUsPageSettings, ActionsPageSettings, ContactUsPageSettings, DonatePageSettings, HomePageSettings, ImpactPageSettings, TeamsPageSettings, Goal, CommunityAdminGroup
+from database.models import Community, CommunityMember, UserProfile, Action, Event, Graph, Media, ActivityLog, AboutUsPageSettings, ActionsPageSettings, ContactUsPageSettings, DonatePageSettings, HomePageSettings, ImpactPageSettings, TeamsPageSettings, Goal, CommunityAdminGroup, Location, RealEstateUnit
 from _main_.utils.massenergize_errors import MassEnergizeAPIError, InvalidResourceError, ServerError, CustomMassenergizeError
 from _main_.utils.massenergize_response import MassenergizeResponse
 from _main_.utils.context import Context
 from django.db.models import Q
-from .utils import get_community_or_die, get_user_or_die, get_new_title
-import random 
+from .utils import get_community_or_die, get_user_or_die, get_new_title, is_reu_in_community
+import random
+import zipcodes
 from sentry_sdk import capture_message, capture_exception
 
 class CommunityStore:
   def __init__(self):
     self.name = "Community Store/DB"
 
+  # QUESTION FOR SAM:
+  # These helper routines don't make use of self, should they be member functions as currently implemented, 
+  # or outside the CommunityStore class?
+
+  def _check_geography_unique(self, community, geography_type, loc):
```

Comment:
nit: maybe `geography_is_unique` is better.  Also for function specifications use """ .....""" instead of #

Reply:
maybe you like another name better, but that is style, not substance.